markdown/pxf/PXFExternalTableandAPIReference.html.md.erb

```diff
@@ -232,23 +250,23 @@ public class InputData {
 
 ### <a id="fragmenter"></a>Fragmenter
 
-**Note:** The Fragmenter Plugin reads data into HAWQ readable external tables. The Fragmenter Plugin cannot write data out of HAWQ into writable external tables.
+**Note:** The Fragmenter class reads data into HAWQ readable external tables. The Fragmenter class cannot write data out of HAWQ into writable external tables.
```

Comment:
It seems like "reads" in this sentence should be changed to "formats" or "loads", but maybe neither of those are correct.  You decide.